docs/CMIP7/guidance_for_users.md

```diff
@@ -5,11 +5,233 @@ title: CMIP7 Guidance for Data Users
+
+To enable modeling groups and others who support CMIP7 to demonstrate its impact (and secure ongoing funding), you are required to cite and acknowledge those who have made CMIP7 possible. When using CMIP7 data, you must: 
+
+ 1. **Acknowledge CMIP7.**
+
+    In the Acknowledgment section, please insert the following text:
+
+    >We acknowledge the World Climate Research Programme, which, through its Working Group on Coupled Modelling, coordinated and promoted CMIP7. We thank the climate modeling groups for producing and making available their model output, the Earth System Grid Federation (ESGF) for archiving the data and providing access, and the multiple funding agencies who support CMIP7 and ESGF.
+
+ 2. **Cite the specific dataset(s) used.**
+
+    Please include a citation in the form of:
+
+        Authors/Data Creators (publication year): Title. Version YYYYMMDD. Earth System Grid Federation. DOI.
+        
+    e.g. 
+
+        Swart et al. (2019): CCCma CanESM5 model output prepared for CMIP6 ScenarioMIP. Version 20190429. Earth System Grid Federation. https://doi.org/10.22033/ESGF/CMIP6.1317. 
+
+    If the latest dataset version included in your study is unknown, use the date of data download instead to characterize the version.
+    If multiple models are used in a publication, please include a table with the sources (name of the model), institutions and citations.
+
+    ??? Note "How to find the DOI?"
+
+        The DOIs can be found through the [Citation Search][citesearch] or in the citation tab of a dataset on MetaGrid.
+
+        It is also possible to take the `tracking_id` global attribute of a file and append it to [https://handle-esgf.dkrz.de/lp/](https://handle-esgf.dkrz.de/lp/) (e.g., [https://handle-esgf.dkrz.de/lp/21.14100/3c2cf1eb-921d-3f48-b0c2-982ef55d93d4](https://handle-esgf.dkrz.de/lp/21.14100/3c2cf1eb-921d-3f48-b0c2-982ef55d93d4)). The DOI will appear on the page of the handle service.
+
+        Note that there are two citation granularities on experiment data and on model/MIP data.
+
+        Further information on the data citation concept for CMIP7 is available [here][cmipcite] and described in [Stockhause and Lautenschlager (2017)][Stockhause2017]. Citations can also be search using [DataCite's catalog][datacitecat] and [Google's Dataset Search][gdatasetsearch].
+    
+ 3. **Cite a paper from the GMD special issue**
+
+    Cite, as appropriate, one or more of the [CMIP7 GMD special issue][GMDSpecialIssue] articles, which include an overview of the CMIP7 experiment design and descriptions of the CMIP7 endorsed MIPs.
+    TODO: should we be clearer ? ask to cite Dunne specifically ?
+ 
+ 4. **Register your work.**
+
+    Register your work on the [CMIP7 Publication Hub][CMIPpubs]. 
+    TODO:do we still want this ?
+
+5. **Adhere to the license**
+    
+    Adhere to licensing restrictions associated with the Creative Commons Attribution 4.0 International ([CC BY 4.0]) license.
```

Comment:
Given groups could instead choose the CC0 1.0 Universal licence, should we just write "Adhere to the license conditions" and not specify only CC BY 4.0 ?